.envs/.production/.stupoli

```diff
@@ -0,0 +1,33 @@
+# General
+# ------------------------------------------------------------------------------
+DJANGO_SETTINGS_MODULE=config.settings.production
+
+DJANGO_SECRET_KEY='uF0sHMCxlAmPJlxpIY8QLUJElAWigxVdOwC92qe0l7Q7LBMfXNWSWQWxV34D8x53'
```

Comment:
this value needs to be in a .secrets file that is gitignored so it does not appear in git.